peopleops/recruiting/HIRING_FLOW.md

```diff
@@ -0,0 +1,81 @@
+# Hiring Flow
+
+This is a suggested hiring flow to make your life easier and standardize our process. On this case, I'm using Upwork platform to hire temporary staff, but you can adapt this system according to your needs.
+
+### Pre-requisites tools
+
+1) Download Upwork app to receive instant notifications from potential leads when they answer to your offer. It's an important step because most freelancers are in different timezones, so you'll have a huge gap in communication if you don't respond to them right away.
+
+2) Use a timezone conversion system like [***Savvy Time***](https://savvytime.com/converter/pdt-to-ist), to schedule the meetings that should occur after the initial screening.
+
+3) Use any video-chat service, like [Appear.in](http://appear.in/countable) to schedule an interview with your best candidates.
+
+4) You can check for more tools at our [peopleops folder](https://drive.google.com/drive/folders/1HY7LPI_h5F5WxaTW5v-1_DMUXcJIjjh1)
+
+## Initial Screening
+
+1) Create an account at ***Upwork*** and click on the CTA "Post a Job" to create a post according to your needs. Make sure you include all relevant keywords in your new ad since it will improve your lead capturing.
+
+2) Once it's created, check it by clicking at the navbar: JOBS > MY JOBS. It will appear at "My Postings" section, where you can see how many proposals you received and how many people messaged you.
+
+3) After some time you'd receive your first leads if you've set your job post correctly.
+     - If not, delete it and create another, making sure you're entering relevant keywords to it. If you're unsure about the job requirements, ask your supervisor's assistance.
+
+4) At "REVIEW PROPOSALS" step, click on "Filters". Now you should ***shortlist***(click on the thumbs-up icon) the candidates that pass the following suggested minimum criteria:
+ - Earned amount > 10k
+ - Job success > 90%
+ - Hours billed: 100+
+- English level: conversational
+- Talent type: freelancers & agencies
+
+5) Now send a message to the selected ones, asking for a quick (15 min) video chat interview. Check what timezone he's in, and use a [***time zone conversion tool***](https://savvytime.com/converter/pdt-to-ist) to make sure it will match yours.
+
+6) It's important to remember scheduling the interview time in an Alarm app (any app that remembers you about it) since once you get 2+ interviews on the same day, you'll have a hard time trying to remember when exactly did you scheduled what meeting.  Be careful to not confusing timezones!
+
+## Scheduling
+
+- Please use [Calendly](https://calendly.com/) to schedule your meetings and avoid timezone issues.
+
```

Comment:
Could also ask the other person to schedule the interview to see if they have those skills. Thoughts?

Reply:
I think it will only add "friction" to the process... It's better to test them directly on what we need. We don't want to lose a good candidate due to timezone issues

Reply:
Ok, although it seems like part of what makes a candidate "good" is their ability to be organized and not waste our team's time. We should assume whatever they do in the application process they'll do on the job. Communication skills are just as critical as technical skills.